### PR TITLE
Data model schema for AgriGreenhouse

### PR DESCRIPTION
**Pull request by fdelavega**
Schema for AgriGreenhouse data model

### CODE REVIEW

File: specs/AgriFood/AgriGreenhouse/example-normalized-ld.jsonld [hunk 1]
```diff
@@ -0,0 +1,79 @@
+{
+    "@context": [
+        "https://forge.etsi.org/gitlab/NGSI-LD/NGSI-LD/raw/master/coreContext/ngsi-ld-core-context.json",
```
Comment (jmcanterafonseca) [inline specs/AgriFood/AgriGreenhouse/example-normalized-ld.jsonld]: use the official @context

File: specs/AgriFood/AgriGreenhouse/example-normalized-ld.jsonld [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        "https://raw.githubusercontent.com/GSMADeveloper/NGSI-LD-Entities/master/examples/Agri-Greenhouse-context.jsonld"
+    ],
+    "id": "urn:ngsi-ld:AgriGreenhouse:df72dc57-1eb9-42a3-88a9-8647ecc954b4",
+    "type": "AgriGreenhouse",
+    "createdAt": "2017-01-01T01:20:00Z",
+    "modifiedAt": "2017-05-04T12:30:00Z",
+    "source": "https://source.example.com",
+    "dataProvider": "https://provider.example.com",
+    "entityVersion": 2.0,
```
Comment (jmcanterafonseca) [inline specs/AgriFood/AgriGreenhouse/example-normalized-ld.jsonld]: check comments on the initial PR

File: specs/AgriFood/AgriGreenhouse/schema.json [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+		},
+		"relativeHumidity": {
+		    "type": "object",
+		    "values": {
+			"value": {
+			    "type": "number",
+			    "minimum": 0.0,
+			    "maximum": 1.0
+			},
+			"unitCode": {
```
Comment (jmcanterafonseca) [inline specs/AgriFood/AgriGreenhouse/schema.json]: be careful, not needed in the base Schema

File: specs/AgriFood/AgriGreenhouse/schema.json [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+			    "type": "string"
+			},
+			"observedAt": {
+			    "type": "string",
+			    "format": "date-time"
+			}
+		    },
+		    "required": ["value", "unitCode", "observedAt"]
```
Comment (jmcanterafonseca) [inline specs/AgriFood/AgriGreenhouse/schema.json]: not needed

File: specs/AgriFood/AgriGreenhouse/schema.json [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+			"unitCode": {
+			    "type": "string",
+			    "enum": ["CEL"]
+			},
+			"observedAt": {
+			    "type": "string",
+			    "format": "date-time"
+			}
+		    },
+		    "required": ["value", "unitCode", "observedAt"]
```
Comment (jmcanterafonseca) [inline specs/AgriFood/AgriGreenhouse/schema.json]: same comment

File: specs/AgriFood/AgriGreenhouse/example-normalized-ld.jsonld (hunk 1 above)
Comment (samgh96) [reply specs/AgriFood/AgriGreenhouse/example-normalized-ld.jsonld]: I used the NGSI-LD example in https://github.com/GSMADeveloper/NGSI-LD-Entities/blob/master/definitions/Agri-Greenhouse.md

File: specs/AgriFood/AgriGreenhouse/schema.json [hunk 6]
```diff
@@ -0,0 +1,83 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$id": "https://raw.githubusercontent.com/Ficodes/dataModels/agriDataModels/specs/AgriFood/AgriGreenhouse/schema.json",
```
Comment (jmcanterafonseca) [inline specs/AgriFood/AgriGreenhouse/schema.json]: use the FIWARE URL

File: specs/AgriFood/AgriGreenhouse/schema.json [hunk 7]
```diff
@@ -0,0 +1,83 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "$id": "https://raw.githubusercontent.com/FIWARE/dataModels/agriDataModels/specs/AgriFood/AgriGreenhouse/schema.json",
```
Comment (jmcanterafonseca) [inline specs/AgriFood/AgriGreenhouse/schema.json]: you don't need such a long URL, please check the other FIWARE Data Models.

This comment applies for the rest of PRs

**Pull request merged**